src/pyggdrasil/visualize/_mcmc.py

```diff
@@ -1,124 +1,122 @@
 """Methods for visualizing mcmc samples."""
```

Comment:
As expected, I largely rewrote this entire file. - Learned how to modularize plotting functions in the last PR though. - Was still very useful.